data/constants/keycodes/keycodes_0.0.1_quantum.hjson

```diff
@@ -220,6 +220,13 @@
                 "UC_EMAC"
             ]
         },
+        "0x7C38": {
```

Comment:
?? That's not how keycode versioning works.

Reply:
Specifically, this should be reverted.

A `keycodes_0.0.8.hjson` and `keycodes_0.0.8_quantum.hjson` would need to be created.

Reply:
@drashna thanks for the clear hint! Please check it out now

Reply:
I ran `util/regen.sh` to regenerate  keycodes. Is that it or something else is needed?